### PR TITLE
Update token binding type attribute in Application REST API

### PR DESCRIPTION
**Pull request by pamodaaw**
Fix https://github.com/wso2/product-is/issues/8613

### CODE REVIEW

File: components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml [hunk 1]
```diff
@@ -2800,6 +2794,12 @@ components:
           type: integer
           format: int64
           example: 3600
+        bindingType:
+          type: string
+          description: Access token binding type.
```
Comment (mefarazath) [inline components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml:2799]: Can we improve the description to say what access token binding is actually?
Basically to give the API consumer an idea as to why he should use this and what he can do with this?

File: components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml [hunk 2]
```diff
@@ -2800,6 +2794,12 @@ components:
           type: integer
           format: int64
           example: 3600
+        bindingType:
```
Comment (mefarazath) [inline components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml:2797]: Can we set the default value to 'None'?

We can either do this at the API swagger or at the code.

Comment (pamodaaw) [reply components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml:2797]: Updated with https://github.com/wso2/identity-api-server/pull/186/commits/9a85e7ed286f2a7422abc510521aaa02ccf0fd4a

File: components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml (hunk 1 above)
Comment (pamodaaw) [reply components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml]: Updated with https://github.com/wso2/identity-api-server/pull/186/commits/9a85e7ed286f2a7422abc510521aaa02ccf0fd4a

**Pull request merged**